examples/local/port_fixed_ip.yaml

```diff
@@ -0,0 +1,112 @@
+tosca_definitions_version: cloudify_dsl_1_3
```

Comment:
More important is to test this on a manager.

Reply:
Sure will give a try there